tests/oauth2/rfc6749/test_utils.py

```diff
@@ -108,3 +109,19 @@ def test_scope_to_list(self):
 
 
 
+class TestUrlValidation(TestCase):
+    def test_basic_urls(self):
```

Comment:
Can this be parameterized to be several different test cases, rather than one test that makes multiple assertions?